### PR TITLE
Update network.md

### PR DESCRIPTION
**Pull request by bax1111**
The connection profile name above is "my-network" not "HassOS default"

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @bax1111, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;operating-system#1197).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: Documentation/network.md [hunk 1]
```diff
@@ -123,13 +123,13 @@ homeassistant login:
 
 From there you use the `nmcli` configuration tool.
 
-- `# nmcli con show` will list the "HassOS default" connection in use.
-- `# nmcli con show "HassOS default"` will list all the properties of the connection.
+- `# nmcli con show` will list the "my-network" connection in use.
+- `# nmcli con show "my-network"` will list all the properties of the connection.
```
Comment (agners) [inline Documentation/network.md:127]: Not sure why this change is needed? On my end the default network name in a newly installed system is `HassOS default`:

```
# nmcli con show
NAME            UUID                                  TYPE      DEVICE 
HassOS default  f62bf7c2-e565-49ff-bbfc-a4cf791e6add  ethernet  eth0   
```

Comment (pvizeli) [reply Documentation/network.md:127]: I agree. Could be that very old installation are not like this. Anyway today we have an UI for this, maybe we don't need this add all and should promote our CLI

Comment (agners) [reply Documentation/network.md:127]: Ok, that probably would need a new merge request. Closing this one.
